src/OTSession.js

```diff
@@ -67,12 +67,11 @@ export default class OTSession extends Component {
   }
   disconnectSession() {
     OT.disconnectSession((disconnectError) => {
+      const events = sanitizeSessionEvents(this.props.eventHandlers);
+      removeNativeEvents(events);
```

Comment:
@msach22 Those disconnect callback on session and destroy callback on publisher are useless as for me. Also they are error prone. Destruction/cleanup code must be invoked regardless any conditions, otherwise it does not work at all

Reply:
I did remove those callbacks at all in my fork, so the code is just invoked this way
removeNativeEvents(..);
OT.disconnectSession();
Without any conditions and callbacks. There is no way to handle the disconnect error because the component is already unmounted